src/General/Divider/Divider.test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import 'jest-styled-components';
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+
+import Divider from './Divider';
+
+describe('<Divider/> snapshots', () => {
+  test('should match snapshot when no props is being passed', () => {
+    const { asFragment } = render(<Divider />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  test('should match snapshot when all props is being passed', () => {
+    const { asFragment } = render(
+      <Divider theme="blue" className="name" style={{ top: '10px' }} />
```

Comment:
Can we test the other themes too? Needed for 100% coverage. May be done programatically.

Reply:
ok